package.json

```diff
@@ -28,6 +28,9 @@
     "test": "npm-run-all test:*",
     "test:backend": "nyc mocha --reporter=min test/backend",
     "test:frontend": "cross-env NODE_ENV=development node test/frontend/runner.js && nyc report --reporter=html",
+    "test:integration-local": "tox -c test/integration/tox.ini",
+    "test:integration-docker": "docker-compose exec -T --user root selenium-firefox tox -c test/integration/tox.ini",
```

Comment:
Both `test:integration-local` and `test:integration-docker` shouldn't run when `npm test` is run. For now I'd like  neither of them to run on `npm test` until we can go from `npm install` to `npm test` without any intermediate steps. I'm happy to do that later. All this entails is renaming them to not start with `test:`

Reply:
Something like ```test-``` ?

Reply:
sure